go.mod

```diff
@@ -10,12 +10,12 @@ replace (
 
 require (
 	github.com/go-logr/logr v0.4.0
-	github.com/layer5io/meshkit v0.5.8
+	github.com/layer5io/meshkit v0.5.12
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.18.1
-	k8s.io/api v0.22.4
-	k8s.io/apimachinery v0.22.4
-	k8s.io/client-go v0.22.4
+	k8s.io/api v0.23.0-alpha.1
```

Comment:
Maybe not a good idea we upgrade the dependencies' version to the beta version. I suggest to we can lock the dependencies to specific version because `meshkit` is a tool packages.